### ISSUE TITLE
No alerts following a maintenance window

### ISSUE THREAD

**Issue by resnostyle** [scobal/seyren#388]
In our subscriptions we have a maintenance window where no alerts should be sent. After that window closes, seyren should alert that the check is in a poor state however it doesn't seem to do that. 

It appears that notifications are only sent on state changes and not reinitialized following a window change.

**Comment by RSamal** [comment]
I agree. At least it should send notification  even if the check continues to be in ERROR state. That would solve the maintenance window issue.

**Comment by paulej** [comment]
I was just looking at that logic. It appears that the code cycles through subscriptions at the time the "check" status is changed and only sends out notifications if that subscription is active at that time a notification is warranted:
https://github.com/scobal/seyren/blob/master/seyren-core/src/main/java/com/seyren/core/service/schedule/CheckRunner.java#L135

One could save the notifications not dispatched and then pick those up on the next check.  But, it would be important not to send out stale notifications.  I agree, though, that it would be much better to receive those notifications once the window opens.
